### PR TITLE
Multiple update tags

### PR DESCRIPTION
**Pull request by zurfyx**
With `history-merge` being a sort of imposed tag to follow certain behavior, you're forced to trigger another `editor.update` to pass your own tag. This is a patchy workaround. What's more, this workaround doesn't work for `discrete` updates.

This diff adds support for an array of tags, I kept the `tag` name singular for backward compatibility.

### CODE REVIEW

**Comment by etrepum** [comment]
FWIW there is `$addUpdateTag` which can be used instead of triggering multiple `editor.update` calls (or using the tag property at all)

**Review by etrepum — APPROVED** [review]
LGTM

File: packages/lexical/src/LexicalUpdates.ts [hunk 1]
```diff
@@ -219,6 +219,20 @@ function $normalizeAllDirtyTextNodes(
   }
 }
 
+function addTags(editor: LexicalEditor, tags: undefined | string | string[]) {
+  if (!tags) {
```
Comment (acywatson) [inline packages/lexical/src/LexicalUpdates.ts:223]: be strict here

**Review by acywatson — APPROVED** [review]

File: packages/lexical/src/LexicalUpdates.ts (hunk 1 above)
Comment (etrepum) [inline packages/lexical/src/LexicalUpdates.ts:223]:
```suggestion
  if (tags === undefined || tags.length === 0) {
```

**Review by etrepum — APPROVED** [review]

**Pull request merged**